dev-lua/luarocks/luarocks-3.3.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A deployment and management system for Lua modules"
+HOMEPAGE="http://www.luarocks.org"
+SRC_URI="http://luarocks.org/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="virtual/pkgconfig"
```

Comment:
as in previous PRs, busted dep is missing.
and you likely  need runtime deps in bdepend for tests.

Reply:
fixed